src/pages/wishlist/index.jsx

```diff
@@ -1,5 +1,66 @@
 import React from "react";
-
+import { Card } from "../../components/Card/Card";
+import { useCart } from "../../context/cart-context";
+import { useWishlist } from "../../context/wishlist-context";
+import wishlistIconFilled from "../../assets/wishlist-filled.png";
+import wishlistIcon from "../../assets/wishlist.png";
+import "./wishlist.css";
 export const Wishlist = () => {
-  return <h3>This is Wishlist</h3>
-}
\ No newline at end of file
+  const {
+    wishlist,
+    addToWishlist,
+    removeFromWishlist,
+    moveToCart,
+    clearWishlist,
+  } = useWishlist();
+  const { items } = wishlist;
+
+  const isInWishlist =(id) => [...items].filter((item) => item["_id"] === id).length > 0
```

Comment:
This can be a generic function that can be used to check for wishlist as well as cart by passing the parameters accordingly